### PR TITLE
Minor Grammatical correction

### PR DESCRIPTION
**Pull request by amalkurup89**
**What this PR does / why we need it**: Minor Grammatical correction

**Which issue(s) this PR fixes**: Documentation fix

### CODE REVIEW

File: docs/sources/tutorials/stack_guide_graphite.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ database has yet to match Graphite's query flexibility and analytics potential.
 Graphite has a reputation for being tricky to install and scale. This guide aims to show
 that is not really the case, or, at least, that it is a lot better than you expect.
 
-> This guide does not only aim to be only be an install guide but to also teach you
+> This guide does not only aim to be an install guide but to also teach you
```
Comment (achatterjee-grafana) [inline docs/sources/tutorials/stack_guide_graphite.md:20]: Please change to: This is an install guide that also teaches you of the mechanics of metric collection ...

Comment (amalkurup89) [reply docs/sources/tutorials/stack_guide_graphite.md]: @achatterjee-grafana Committed the suggested changes. Please approve

**Review by achatterjee-grafana — APPROVED** [review]
Looks good! Thanks.

**Comment by amalkurup89** [comment]
@achatterjee-grafana Can you please merge this PR

**Pull request merged**